README.md

```diff
@@ -32,15 +32,15 @@ To get a local copy up and running follow these simple example steps.
 
 If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
 
```

Comment:
- [ ] Please deploy your app and make sure the live demo is working just fine, then attach it to the README file to get your approval on this. 😉